components/02-components/01-card/01-action-card/action-card.config.json

```diff
@@ -3,9 +3,12 @@
     "name": "action card",
     "status": "prototype",
 	"context": {
-        "headline": "Lorem ipsum dolor sit amet, consectetur",
-		"text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse vel purus ut arcu tempor mollis vitae sit amet nulla. Morbi quis interdum odio, ut hendrerit",
-        "label": "Action"
+		"pure": false,
```

Comment:
I had to add a "pure" property to trigger the handlebars 'IF' helper.

"pure" is a prop name I made up...